src/Components/Cards/Cards.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Card = ({ firstName, lastName, street, unit, city, state, zip, speciality, phoneNum, key, vetted }) => {
```

Comment:
I'm a fan of the destructuring. Looks good! Glad to see our hard-earned data displaying to the page :)